app/src/main/java/chat/rocket/android/contacts/adapter/ContactsRecyclerViewAdapter.kt

```diff
@@ -62,7 +67,8 @@ class ContactsRecyclerViewAdapter(
             is ContactsItemHolder -> VIEW_TYPE_CONTACT
             is ContactsHeaderItemHolder -> VIEW_TYPE_HEADER
             is InviteItemHolder -> VIEW_TYPE_INVITE
-            else -> throw IllegalStateException("View type must be either Room, Header or Invite")
+            is PermissionsItemHolder -> VIEW_TYPE_PERMISSIONS
+            else -> throw IllegalStateException("View type must be either Room, Header, Invite or Permissions.")
```

Comment:
technically wouldn't these be "Contact" instead of "Room"?

Reply:
done